qemu/tests/cfg/ioeventfd.cfg

```diff
@@ -1,11 +1,19 @@
 - ioeventfd: install setup image_copy unattended_install.cdrom
     type = ioeventfd
-    only virtio_blk virtio_scsi
+    only virtio_blk virtio_scsi virtio_serial
```

Comment:
We only put the options for one same `variants` together.
Move `virtio_serial` to a separated line.

Reply:
And, where is `virtio_serial` ?

Reply:
Thanks and update, and will delete `virtio_serial`.